backend/src/main/java/com/handong/rebon/exception/ExceptionResponse.java

```diff
@@ -0,0 +1,2 @@
+package com.handong.rebon.exception;public class ExceptionResponse {
+}
```

Comment:
pr전에 포맷팅 한 번 적용하면 좋을 것 같아요.
다른 exception들도 한 번씩 확인해 보면 좋을 것 같네요

```suggestion
package com.handong.rebon.exception;

public class ExceptionResponse {
}
```

Reply:
제가 코드들을 다 push해서 올리지 않았었네요.....!. 이런 실수해서 죄송합니다! 깃 크라켄 쓰다가 적용이 잘못된줄 알고 안올렸나봐요! 수정하겠습니다